src/chrome/content/rules/DigiBib.net.xml

```diff
@@ -9,10 +9,10 @@
 -->
 <ruleset name="DigiBib.net">
 
-	<target host="*.digibib.net" />
+	<target host="m.digibib.net" />
+	<target host="www.digibib.net" />
```

Comment:
@Bisaloo I have modified the code and the domains are sorted alphabetically now. I do not want to move `www` to the top because `^` is always missing in this PR.